README.md

```diff
@@ -52,6 +52,23 @@ ax.plot(a)
 plt.show()
 ```
 
+## `generate_hex_colors`
+
+```python
+import matplotlib.pyplot as plt
+import cosmoplots
+
+color_list = cosmoplots.generate_hex_colors(5, 'viridis', show_swatch=True, ascending=True)
+plt.savefig("/assets/color_test.png")
```

Comment:
Missing a `.` in the path: -> `./assets/color_test.png`.

Reply:
Thanks for spotting. Done!